notes/WORKSPACES_CMDS.md

```diff
@@ -40,6 +40,10 @@ Should trim loadActual tree that starts at the node `./workspace-a/`.
 
 Sets Arb.path to point to the workspace path.
 
+### npm init
+
+Should it create a new workspace?
```

Comment:
what does this mean? a new workspace subproject? is a "workspace" the root, or a project within a workspace?

or are you asking if `npm init` should ask "is this meant to be a workspace"?

Reply:
The list of command here is meant to be collecting thoughts on the greater question: "What should the workspaces-aware version of a command do?" - Sorry this is not super well formatted and is more of a brainstorm dump 😬 

But the more fundamental question this is trying to answer is: How does a user initialize a new workspace in the current project?

Reply:
maybe i'm unclear on the terms. the directory itself with the package.json (the repo) is the "workspace", no?

Reply:
hehehe I feel like I failed very hard on my mission of trying to improve on the confusing terms situation but to answer to your question: NO, "workspace" here refers to the nested packages and "Top-level workspace" is the term for referring to "the repo", ref: https://github.com/npm/rfcs/blob/cb9ca1459108eadbaa2a94372fbec533450e1584/accepted/0000-workspaces-part2.md#dictionary